docs/reference/UDS Core/prerequisites.md

```diff
@@ -136,3 +140,180 @@ Metrics server is provided as an optional component in UDS Core and can be enabl
+          - path: loki.storage.s3.region
+            value: ""
+          - path: loki.storage.s3.accessKeyId
+            value: ${LOKI_ACCESS_KEY_ID}
+          - path: loki.storage.s3.secretAccessKey
+            value: ${LOKI_SECRET_ACCESS_KEY}
+          - path: loki.storage.s3.s3ForcePathStyle
+            value: true
+          - path: loki.storage.s3.signatureVersion
+            value: "v4"
+          - path: write.extraArgs
+            value:
+            - "-config.expand-env=true"
+          - path: write.extraEnv
+            value:
+            - name: LOKI_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: minio-loki
+                  key: accessKey
+            - name: LOKI_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: minio-loki
+                  key: secretKey
+          - path: read.extraArgs
+            value:
+            - "-config.expand-env=true"
+          - path: read.extraEnv
+            value:
+            - name: LOKI_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: minio-loki
+                  key: accessKey
+            - name: LOKI_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: minio-loki
+                  key: secretKey
+```
+
+#### Velero
+
+The Velero deployment is (by default) backed by an object storage provider for backup retention.  For cloud environments you can wire this into the environment's storage provider with the following overrides:
+
+```yaml
+velero:
+  values:
+    - path: credentials.secretContents.cloud
+      value: |
+        [default]
+        aws_access_key_id=<s3-access-key>
+        aws_secret_access_key=<s3-secret-key>
+    - path: "configuration.backupStorageLocation"
+      value:
+        - name: default
+          provider: aws
+          bucket: "<bucket-name>"
+          config:
+            region: "<s3-region>"
+            s3ForcePathStyle: true
+            s3Url: "<s3-endpoint>"
+          credential:
+            name: "velero-bucket-credentials"
+            key: "cloud"
+```
+
+You can also use the [Minio Operator UDS Package](https://github.com/defenseunicorns/uds-package-minio-operator) to back Velero with the following overrides:
+
+```yaml
+- name: minio-operator
+  ...
+  overrides:
+    minio-operator:
+      uds-minio-config:
+        values:
+          - path: apps
+            value:
+              - name: velero
+                namespace: velero
+                remoteSelector:
+                  app.kubernetes.io/name: velero
+                bucketNames:
+                  - uds-velero
+                copyPassword:
+                  enabled: true
+                  secretIDKey: AWS_ACCESS_KEY_ID
+                  secretPasswordKey: AWS_SECRET_ACCESS_KEY
+
+- name: core-backup-restore
+  ...
+  overrides:
+    velero:
```

Comment:
Same comment here on the egress policy overrides, see [the values here](https://github.com/defenseunicorns/uds-core/blob/add-docs-s3-lb-prerequisites/src/loki/chart/values.yaml).